.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
-    "editor.tabSize": 2,
+    "editor.tabSize": 4,
```

Comment:
:-1:

Reply:
actually it is based on pylintrc ... but I am flexible. You want 2?

Reply:
¡A mí me daban dos!

Reply:
I personally prefer 4 too ... as this makes for better visibility of blocks and it discurrages deep nesting :)